faunadb-scala/src/main/scala/faunadb/FaunaClient.scala

```diff
@@ -45,13 +45,15 @@ object FaunaClient {
     metrics: MetricRegistry = null,
     queryTimeout: FiniteDuration = null,
     userAgent: String = null,
-    checkNewVersion: Boolean = true): FaunaClient = {
+    checkNewVersion: Boolean = true,
+    customHeaders: Map[String, String] = null): FaunaClient = {
```

Comment:
Avoid `null` as value in Scala. Better to use `Map.empty`:
`customHeaders: Map[String, String] = Map.empty`